generator/mavgen_typescript.py

```diff
@@ -74,40 +76,55 @@ def generate_classes(dir, registry, msgs, xml):
 
                 f.write("/*\n{}\n*/\n".format(m.description.strip()))
                 for field in m.fields:
-                    f.write("// {} {} {}\n".format(field.name, field.description.strip(), field.type))
+                    f.write("/* {} {} {} */\n".format(field.name, field.description.strip(), field.type))
```

Comment:
I fields comments should be moved at line 85/86, otherwise they won't work

Reply:
I'm not sure I understand this comment?When I look at lines 84/85  I don't see any comments. What are you proposing?

Reply:
Just to move the comments from here:
```ts
/*
TC ping (expects an ACK message as a response)
*/
/* timestamp Timestamp to identify when it was sent uint64_t */
export class PingTc extends MavLinkData {
	public timestamp!: number;
	...
```

to here:
```ts
/*
TC ping (expects an ACK message as a response)
*/
export class PingTc extends MavLinkData {
        /* Timestamp to identify when it was sent uint64_t */
	public timestamp!: number;
	...
```
Also notice how I removed the field name from the comment, as it is not really necessary.

Reply:
Updated to include this